python/GangaDirac/Lib/Files/DiracFile.py

```diff
@@ -463,7 +463,7 @@ def location(self):
                     LFNs.append(this_url)
             return LFNs
 
-    def accessURL(self, thisSE=''):
+    def accessURL(self, thisSE='', protocol='xroot'):
```

Comment:
We don't want this to be the case for GridPP, smaller VOs (non-CERN VOs) don't use ROOT. Can we make this None by default and check against the config system to get the value correct value (for LHCb it is xroot).